app/models/activity.rb

```diff
@@ -15,8 +15,11 @@ def self._build_object(data)
     activity.start_date = Time.new(data["start_date"][0..3],
                                    data["start_date"][5..6],
                                    data["start_date"][8..9])
+    activity.raw_time = data['elapsed_time']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.